### PR TITLE
Fix grammar in HTML label element

### PR DESCRIPTION
**Pull request by CliqLabs**
Made some minor grammar fixes to improve clarity.

### Description
Made a couple of minor grammar fixes to improve clarity.

### Motivation
Confusing sentence structures make concepts more difficult to understand.  I've reworded a few sentences so they're easier to parse.

### CODE REVIEW

File: files/en-us/web/html/element/label/index.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ There are no special styling considerations for `<label>` elements — structura
 
 ### Interactive content
 
-Don't place interactive elements such as {{HTMLElement("a", "anchors")}} or {{HTMLElement("button", "buttons")}} inside a `label`. Doing so makes it difficult for people to activate the form input associated with the `label`.
+Do not place interactive elements such as {{HTMLElement("a", "anchors")}} or {{HTMLElement("button", "buttons")}} inside a `label`. Doing so makes it difficult to activate the form input associated with the `label`.
```
Comment (Josh-Cena) [inline files/en-us/web/html/element/label/index.md:83]: No need to expand contractions: they are generally encouraged in technical writing to make it sound casual.

File: files/en-us/web/html/element/label/index.md [hunk 2]
```diff
@@ -13,10 +13,10 @@ The **`<label>`** [HTML](/en-US/docs/Web/HTML) element represents a caption for
 
 Associating a `<label>` with a form control, such as {{htmlelement("input")}} or {{htmlelement("textarea")}} offers some major advantages:
 
-- The label text is not only visually associated with its corresponding text input; it is programmatically associated with it too. This means that, for example, a screen reader will read out the label when the user is focused on the form input, making it easier for an assistive technology user to understand what data should be entered.
+- The label text is not only visually associated with its corresponding text input; it is programmatically associated with it, too. This means that, for example, a screen reader will read out the label when the user is focused on the form input, making it easier for an assistive technology user to understand what data should be entered.
```
Comment (Josh-Cena) [inline files/en-us/web/html/element/label/index.md:16]:
```suggestion
- The label text is not only visually associated with its corresponding text input; they are programmatically associated, too. This means that, for example, a screen reader will read out the label when the user is focused on the form input, making it easier for an assistive technology user to understand what data should be entered.
```

**Review by Josh-Cena — COMMENTED** [review]
More drive-by

File: files/en-us/web/html/element/label/index.md [hunk 3]
```diff
@@ -13,10 +13,10 @@ The **`<label>`** [HTML](/en-US/docs/Web/HTML) element represents a caption for
...
 - When a user clicks or touches/taps a label, the browser passes the focus to its associated input (the resulting event is also raised for the input). That increased hit area for focusing the input provides an advantage to anyone trying to activate it — including those using a touch-screen device.
 
-To explicitly associate a `<label>` element with an `<input>` element, you first need to add the `id` attribute to the `<input>` element. Next, you add the `for` attribute to the `<label>` element, where the value of `for` is the same as the `id` in the `<input>` element.
+To explicitly associate a `<label>` element with an `<input>` element, you first need to add the `id` attribute to the `<input>` element. Next, add the `for` attribute to the `<label>` element, where the value of `for` is the same as the `id` in the `<input>` element.
```
Comment (CBID2) [inline files/en-us/web/html/element/label/index.md:19]:
```suggestion
To explicitly associate a `<label>` element with an `<input>` element, you first need to add the `id` attribute to the `<input>` element. Next, add the `for` attribute to the `<label>` element. Lastly, make the value of `for` the same as the `id` in the `<input>` element.
```

**Review by CBID2 — COMMENTED** [review]
Hi @CliqLabs! I just made a suggestion to the tutorial on working with`<label>` and `for` elements. I hope it helps! 😄

**Comment by hamishwillee** [comment]
Hi @CliqLabs - there are some drive-by comments from Josh above - can you please check them?

**Comment by bsmth** [comment]
Thanks for the submission @CliqLabs - I'm going to close this one due to inactivity and unaddressed review comments. Let us know if you'd like to come back to this and we can reopen it.

Thank you :)

**Pull request closed without merging**